builtin/credential/github/path_config.go

```diff
@@ -110,8 +126,8 @@ func (b *backend) Config(s logical.Storage) (*config, error) {
 }
 
 type config struct {
-	Org     string        `json:"organization"`
-	BaseURL string        `json:"base_url"`
-	TTL     time.Duration `json:"ttl"`
-	MaxTTL  time.Duration `json:"max_ttl"`
+	Org     string        `json:"organization" structs:"organization" mapstructure:"organization"`
```

Comment:
Yes, we should match so `organization` is good. That said, since the config is being saved as `organization` in the JSON, can you just name the config struct to match? Tab-completion makes it not really matter when writing code so it just ensures no confusion between the struct name and what comes back :-D